melos.yaml

```diff
@@ -42,14 +42,14 @@ scripts:
 
   analyze: >
     melos exec -c 1 --no-private --ignore="*example*" -- \
-      pub global run tuneup check
+      dart pub global run tuneup check
 
   lint:pub: >
     melos exec -c 5 --fail-fast --no-private --ignore="*example*" -- \
-      pub publish --dry-run
+      dart pub publish --dry-run
 
   postbootstrap: >
-    (tuneup --version || pub global activate tuneup) && melos run copy_dummy_config
+    (tuneup --version || dart pub global activate tuneup) && melos run copy_dummy_config
```

Comment:
Thanks for submitting this change. I think it's a good change considering standalone `pub` is indeed deprecated 😅  but I wonder if the change should be using `flutter pub` instead:
1. This is, after all, intended to be used with Flutter specifically rather than a generic Dart project. Though the root directory does not technically contain Flutter-specific code, I would interpret the recommendation per the docs (https://dart.dev/tools/pub/cmd) applicable to this project.
2. The `dart` command does not necessarily need to be exposed to the user's shell.
3. We apparently already have scripts relying on the `flutter` command.

Reply:
I think the trouble is which tool you have used to activate Melos and/or tuneup. I considered using flutter for these, but I can’t remember why I settled on dart instead

Reply:
> I think the trouble is which tool you have used to activate Melos and/or tuneup

Hmm I see. Could you help me understand what you mean by this better so we can attempt to address this more holistically? If the `flutter pub` approach doesn't work, it would be great if we knew why so we can try to shore that up.

Reply:
Oh I mean simply that if you use `dart pub global activate melos` to install that tool, you cannot use `flutter pub global run melos` to run it, and vice versa.

Reply:
Ah. Given that it appears to be an issue with either path chosen, would the arguments for `flutter pub` I pointed out be valid after all? What are your thoughts on perhaps updating the melos section of the [CONTRIBUTING.md](https://github.com/aws-amplify/amplify-flutter/blob/master/CONTRIBUTING.md) to nudge users towards installing via the `flutter` command?